cli/lib/kontena/cli/helpers/exec_helper.rb

```diff
@@ -96,6 +96,12 @@ def websocket_exec_write(ws, msg)
     def websocket_exec_write_thread(ws, tty: nil)
       Thread.new do
         begin
+          if tty
+            console_height, console_width = IO.console.winsize
```

Comment:
Eventually this will also need a `SIGWINCH` signal handler to handle console resize.

Reply:
Edited #2601

Reply:
Filed  #2839 for the missing `SIGWINCH` tty resize functionality in this PR.